mscore/musescore.cpp

```diff
@@ -1309,12 +1309,82 @@ MuseScore::MuseScore()
+    QMap<QString, QString> *systemLang = new QMap<QString, QString>();
+    QString lang = getLocaleISOCode();
+    lang = lang.left(2);
+        
+    QStringList layoutList = *new QStringList();
+    layoutList << "US-QWERTY" << "UK-QWERTY" << "QWERTZ" << "AZERTY" << "BEPO" << "DVORAK" << "OTHER";
+    langToLayout->insert("en", layoutList);
+    layoutList.clear();
+        
+    layoutList << "JIS - Standard Japanese" << "OYAYUBI SHIFUTO" << "OTHER";
+    langToLayout->insert("ja", layoutList);
+    layoutList.clear();
+        
+    layoutList << "T1 - Standard German" << "T2" << "Swiss German" << "OTHER";
+    langToLayout->insert("de", layoutList);
+    layoutList.clear();
+        
+    layoutList << "Belgian French" << "Canadian French" << "French French" << "Swiss French" << "OTHER";
+    langToLayout->insert("fr", layoutList);
+    layoutList.clear();
+        
+    layoutList << "Spanish" << "OTHER";
+    langToLayout->insert("es", layoutList);
+    layoutList.clear();
+    
+    layoutList << "Italian" << "OTHER";
+    langToLayout->insert("it", layoutList);
+    layoutList.clear();
+        
+    layoutList << "Portuguese Brazilian ABNT" << "Portuguese" << "OTHER";
+    langToLayout->insert("pt", layoutList);
+    layoutList.clear();
+        
+    layoutList << "US-International" << "Dutch Layout" << "OTHER";
+    langToLayout->insert("nl", layoutList);
+    layoutList.clear();
+        
+    layoutList << "Dubeolsik (두벌식)" << "Sebeolsik (세벌식)" << "OTHER";
+    langToLayout->insert("ko", layoutList);
+    layoutList.clear();
+        
+    layoutList << "US-QWERTY" << "UK-QWERTY" << "QWERTZ" << "AZERTY" << "BEPO" << "DVORAK" << "JIS - Standard Japanese" << "OYAYUBI SHIFUTO" << "T1 - Standard German" << "T2" << "Swiss German" << "Belgian French" << "Canadian French" << "French French" << "Swiss French" << "Spanish" << "Italian" << "Portuguese Brazilian ABNT" << "Portuguese" << "US-International" << "Dutch Layout" << "Dubeolsik (두벌식)" << "Sebeolsik (세벌식)";
+    langToLayout->insert("OTHER", layoutList);
+        layoutList.clear();
+        
+    systemLang->insert("en", "English");
+    systemLang->insert("ja", "Japanese");
+    systemLang->insert("de", "German");
+    systemLang->insert("fr", "French");
+    systemLang->insert("it", "Italian");
+    systemLang->insert("pt", "Portuguese");
+    systemLang->insert("nl", "Dutch");
+    systemLang->insert("ko", "Korean");
+        
+    QString labelMessage = "Detected Language: " + systemLang->values(lang)[0] + ". Enter your keyboard layout:";
+        
+    QString result = QInputDialog::getItem(new QWidget(), "Startup Wizard", labelMessage, langToLayout->values(lang)[0]);
+    if (result == "OTHER") {
+        lang = "OTHER";
+        labelMessage = "Choose keyboard layout:";
+        QString result = QInputDialog::getItem(new QWidget(), "Startup Wizard", labelMessage, langToLayout->values(lang)[0]);
+    }
```

Comment:
Please use banner style braces, closing brace indented the same as preceeding block